### PR TITLE
Bump everything to psammead-styles 1.2.0

### PR DESCRIPTION
**Pull request by andrew-nowak**
Resolves #NUMBER

**Overall change:** Brings in the `sans-serif` fixes and the new `@supports ${grid}` stuff

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-inline-link/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0 | [PR#1365](https://github.com/bbc/psammead/pull/1365) Bump psammead-styles to 1.2.0 |
```
Comment (ghost) [inline packages/components/psammead-inline-link/CHANGELOG.md]: Why are we bumping from `alpha` to `stable` version

**Review by ghost — COMMENTED** [review]
Nice work. The rest looks fine.

File: packages/components/psammead-inline-link/CHANGELOG.md (hunk 1 above)
Comment (andrew-nowak) [reply packages/components/psammead-inline-link/CHANGELOG.md]: ! Great spot, thanks!

**Review by j-pendlebury — APPROVED** [review]

File: packages/components/psammead-inline-link/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-inline-link",
-  "version": "1.2.0-alpha.1",
+  "version": "1.2.1-alpha.1",
```
Comment (dr3) [inline packages/components/psammead-inline-link/package.json]: Should this not be `1.2.0-alpha.2`?

Comment (dr3) [reply packages/components/psammead-inline-link/package.json]: Or at least `1.2.1-alpha.0`

Comment (andrew-nowak) [reply packages/components/psammead-inline-link/package.json]: would agree, but I was just trying to fit in with the previous version bumping which has been doing this style https://github.com/bbc/psammead/pull/1365/files#diff-77e5a30c00e558e1ef1b46ece522934dR8

**Comment by ghost** [comment]
@aacn500 Resolve conflicts.

**Review by sareh — APPROVED** [review]
👍 Looks great!

**Review by ghost — APPROVED** [review]

**Comment by sareh** [comment]
Note for testers: this update means that these changes will be included in the packages:

|Version | Description|
|--|--|
| 1.2.0 | [PR#1234](https://github.com/bbc/psammead/pull/1234) Add custom CSS Grid feature detector |
| 1.1.5 | [PR#1215](https://github.com/bbc/psammead/pull/1215) Fix `sans serif` typo in Persian Nassim font-family |
| 1.1.4 | [PR#1066](https://github.com/bbc/psammead/pull/1066) Fix Serif Medium font to return Medium instead of Italic. |

**Comment by david-boydell** [comment]
Hiya, this looks good and is ready for merge.

**Pull request merged**